x/dispute/keeper/msg_update_team.go

```diff
@@ -31,5 +34,51 @@ func (k msgServer) UpdateTeam(ctx context.Context, msg *types.MsgUpdateTeam) (*t
 	if err := k.Params.Set(ctx, param); err != nil {
 		return nil, err
 	}
+	// if the team has voted on a dispute, transfer vote to the new address
+	iter, err := k.Disputes.Indexes.OpenDisputes.MatchExact(ctx, true)
+	if err != nil {
+		return nil, err
+	}
+	defer iter.Close()
+	for ; iter.Valid(); iter.Next() {
+		key, err := iter.PrimaryKey()
+		if err != nil {
+			return nil, err
+		}
+		dispute, err := k.Disputes.Get(ctx, key)
+		if err != nil {
+			return nil, err
+		}
+		// if dispute is open, check if team has voted
+		if dispute.Open {
+			id := dispute.DisputeId
+			currentAccBytes := currentAcc.Bytes()
+			newAccBytes := newAcc.Bytes()
+			teamVoteExists, err := k.Voter.Has(ctx, collections.Join(id, currentAccBytes))
+			if err != nil {
+				if !errors.Is(err, collections.ErrNotFound) {
```

Comment:
unnecessary check for this error. only returns an error if key is nil which is a different error.
https://github.com/cosmos/cosmos-sdk/blob/8bfcf554275c1efbb42666cc8510d2da139b67fa/collections/map.go#L98-L104